gst/nnstreamer/elements/gsttensor_if.c

```diff
@@ -345,6 +345,49 @@ gst_tensor_if_set_property_glist (const GValue * value, GList ** prop_list,
   g_strfreev (strv);
 }
 
+/**
+ * @brief Convert GValue to GList for cv option
+ */
+static void
+gst_tensor_if_set_property_cv_option (const GValue * value, GList ** prop_list)
+{
+  gint64 val;
+  gint length, i;
+  const gchar *param = g_value_get_string (value);
+  gchar **strv = g_strsplit_set (param, ",", -1);
```

Comment:
we need to use g_strfreev to free strv.
g_strfreev (strv);

Reply:
Added `g_strfreev (strv);`
Thank you :)